cmd/marker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos:centos7
```

Comment:
Shouldn't we stick to the same base system (fedora) that other kubevirt projects use? This duplicates disk size of images.

Reply:
cmd/plugin/Dockerfile currently uses centos7, but I see no reason not to use fedora. Opened an issue for it https://github.com/kubevirt/ovs-cni/issues/57

Reply:
Thanks! And then I agree it doesn't make sense to switch in this PR.